posts/zzz_DO_NOT_EDIT_admiral_1.3.../admiral_1.3.0:_documentation_go!!.qmd

```diff
@@ -0,0 +1,163 @@
+
+- admiral::derive_extreme_records()
+- admiral::derive_param_tte()
+- admiral::derive_summary_records()
+- admiral::derive_extreme_event()
+- admiral::derive_vars_joined()
+- admiral::derive_vars_joined_summary()
+- admiral::derive_vars_merged()
+- admiral::derive_vars_crit_flag()
+- admiral::derive_var_trtemfl()
+- admiral::derive_param_computed()
+- admiral::derive_vars_dt()
+- admiral::derive_vars_dt()
+- admiral::derive_var_extreme_flag()
+
+This is not to say that the other functions do not have examples anymore; rather, they retain the previous material. 
+
+Now, our aim as a development team is to continually enhance the reference examples of our large portfolio of functions regularly across release cycles. If you'd like to play a part, this is also a great opportunity to contribute to {admiral} yourself - please pick an {admiral} function you'd like to enhance and make an issue in our Github repository, tagging it with the "Documentation" and "Examples Enhancement" labels.
+
+We've also made various other small documentation enhancements, which you can peruse in the [Changelog](https://pharmaverse.github.io/admiral/main/news/index.html).
+
+## Other new features
+
+But we haven't stopped there! Here's what else is new in {admiral} 1.3:
+
+- We've enhaned our catalogue of functions with a new addition: admiral::derive_vars_joined_summary(). This function is similar to admiral::derive_vars_joined(), but allows the derivation of summary variables (e.g. cumulative sum, etc) rather than variables using a formula. For instance, here's an example derivation in ADAE of `CUMDOSA`, the cumulative dose received up until an adverse event:
+
+```{r, eval = TRUE}
+library(tibble)
+library(dplyr, warn.conflicts = FALSE)
+library(admiral)
+
+adex <- tribble(
+  ~USUBJID, ~ADY, ~AVAL,
+  "1",         1,    10,
+  "1",         8,    20,
+  "1",        15,    10,
+  "2",         8,     5
+)
+
+adae <- tribble(
+  ~USUBJID, ~ADY, ~AEDECOD,
+  "1",         2, "Fatigue",
+  "1",         9, "Influenza",
+  "1",        15, "Theft",
+  "1",        15, "Fatigue",
+  "2",         4, "Parasomnia",
+  "3",         2, "Truancy"
+)
+
+derive_vars_joined_summary(
+  dataset = adae,
+  dataset_add = adex,
+  by_vars = exprs(USUBJID),
+  filter_join = ADY.join <= ADY,
+  join_type = "all",
+  join_vars = exprs(ADY),
+  new_vars = exprs(CUMDOSA = sum(AVAL, na.rm = TRUE))
+)
+```
+
+- We now provide new lab grading metadata specifically tailored for US (Conventional) units across three grading criteria: NCI-CTCAEv4 (`atoxgr_criteria_ctcv4_uscv`), NCI-CTCAEv5 (`atoxgr_criteria_ctcv5_uscv`), and `DAIDs` (atoxgr_criteria_daids_uscv).
+- Default values for function arguments are now always displayed in arguments description.
+atoxgr_criteria_daids_uscv (DAIDs criteria)
+
+## Updates of existing functions and bug fixes
```

Comment:
honestly i think we should just say check out the changelog and not list all of this here